packages/docs/src/page-configs/ui-elements/input/examples/Default.vue

```diff
@@ -43,6 +43,36 @@
       placeholder="Readonly"
       readonly
     />
+    <va-input
+      v-model="value"
+      counter
+      class="mb-4"
+    />
+    <va-input
+      v-model="value"
+      :max-length="30"
+      counter
+      class="mb-4"
+    >
+      <template #counter="{ counter, maxLength }">
+        <span>Remaining symbols</span>
+        <b style="margin-left: auto;">{{ `${counter} / ${maxLength}` }}</b>
+      </template>
+    </va-input>
+    <va-input
+      v-model="value"
+      :max-length="30"
+      counter
+      class="mb-4"
+    >
+      <template #counter="{ counter, maxLength }">
```

Comment:
`{ counter, ` here "counter" name is very confusing. Should be boolean, but it's seemingly not. Maybe use `valueLength` or just pass down `value`?